docs/freqai.md

```diff
@@ -0,0 +1,687 @@
+
+    def informative_pairs(self):
+        whitelist_pairs = self.dp.current_whitelist()
+        corr_pairs = self.config["freqai"]["feature_parameters"]["include_corr_pairlist"]
+        informative_pairs = []
+        for tf in self.config["freqai"]["feature_parameters"]["include_timeframes"]:
+            for pair in whitelist_pairs:
+                informative_pairs.append((pair, tf))
+            for pair in corr_pairs:
+                if pair in whitelist_pairs:
+                    continue  # avoid duplication
+                informative_pairs.append((pair, tf))
+        return informative_pairs
+
+    def bot_start(self):
+        self.model = CustomModel(self.config)
+
+    def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
+
+        self.freqai_info = self.config["freqai"]
+
+        # All indicators must be populated by populate_any_indicators() for live functionality
+        # to work correctly.
+
+        # the model will return all labels created by user in `populate_any_indicators` 
+        # (& appended targets), an indication of whether or not the prediction should be accepted, 
+        # the target mean/std values for each of the labels created by user in 
+        # `populate_any_indicators()` for each training period.
+
+        dataframe = self.model.bridge.start(dataframe, metadata, self)
+
+        return dataframe
+```
+
+The user should also include `populate_any_indicators()` from `templates/FreqaiExampleStrategy.py` which builds 
+the feature set with a proper naming convention for the IFreqaiModel to use later.
+
+### Building an IFreqaiModel
+
+FreqAI has multiple example prediction model based libraries such as `Catboost` regression (`freqai/prediction_models/CatboostPredictionModel.py`) and `LightGBM` regression. However, users can customize and create
+their own prediction models using the `IFreqaiModel` class. Users are encouraged to inherit `train()` and `predict()` to let them customize various aspects of their training procedures.
+
+### Running the model live
+
+Freqai can be run dry/live using the following command
+
+```bash
+freqtrade trade --strategy FreqaiExampleStrategy --config config_freqai.example.json --freqaimodel ExamplePredictionModel
+```
+
+By default, Freqai will not find find any existing models and will start by training a new one 
+given the user configuration settings. Following training, it will use that model to predict for the
+duration of `backtest_period_days`. After a full `backtest_period_days` has elapsed, Freqai will auto retrain 
+a new model, and begin making predictions with the updated model. FreqAI backtesting and live both
+permit the user to use fractional days (i.e. 0.1) in the `backtest_period_days`, which enables more frequent 
+retraining. But the user should be careful that using a fractional `backtest_period_days` with a large
+`--timerange` in backtesting will result in a huge amount of required trainings/models.
```

Comment:
This paragraph looks/sounds wrong with the new parameter naming (could be me misunderstanding it though) - and i think should be using `live_retrain_hours` - or `train_period_days`

Reply:
Yes, you are right - this needs to be updated now that we have the new parameter `live_retrain_hours`. Nice catch.